integration-tests/ccip-tests/testsetups/ccip.go

```diff
@@ -1434,3 +1435,11 @@ func createEnvironmentConfig(t *testing.T, envName string, testConfig *CCIPTestC
 	}
 	return envConfig
 }
+
+func SleepAndReplay(t *testing.T, e ccipdeployment.DeployedEnv, sourceChain, destChain uint64) {
```

Comment:
Lets keep the copy/pasted version for now because I see a few problems with this func:

* Only for two chains
* Hardcodes replay blocks to 1

This only applies for a subset of all tests + tests that only run on simulated. We need to review all the replays we do to make sure we replay from a reasonable block value in real testnet cases

Reply:
We should just fix it via https://smartcontract-it.atlassian.net/browse/CCIP-2967